src/masternodes/loan.h

```diff
@@ -0,0 +1,73 @@
+#ifndef DEFI_MASTERNODES_LOAN_H
+#define DEFI_MASTERNODES_LOAN_H
+
+#include <tuple>
```

Comment:
I do not think that tuple, pubkey.h or script.h are used.

Reply:
Copy/paste from ICX.